### ISSUE TITLE
Differing slant values for faux italics compared to VSFilter

### ISSUE THREAD

**Issue by GrygrFlzr** [libass/libass#301]
Real world example: ![Different slant values](https://www.goodjobmedia.com/wp-content/uploads/2018/03/flipflap03tscompare.png)
In this example there were masks on top of a faux-italic font. Since the masks were aligned against vsfilter's italics, it made the issue more obvious.

Reduced test case:
```ass
Dialogue: 0,0:00:00.00,0:00:05.00,Default,,0,0,0,,{\pos(640,360)\fnJunegull\i1\fs150\bord0\shad0\an5}HELLO WORLD
```
Font available [here](https://www.dafont.com/junegull.font).
Comparison [here](https://diff.pics/qurx2WoY9OiQ/1).

From trial-and-error, VSFilter's slant is equivalent to `\fax-0.34` whereas libass' is equivalent to `\fax-0.21` (with the appropriate shifts in position).
vsfilter `\fax`-based equivalent:
```ass
Dialogue: 0,0:00:00.00,0:00:05.00,Default,,0,0,0,,{\pos(681,360)\fnJunegull\fax-0.34\fs150\bord0\shad0\an5}HELLO WORLD
```
libass `\fax`-based equivalent:
```ass
Dialogue: 0,0:00:00.00,0:00:05.00,Default,,0,0,0,,{\pos(665.5,360)\fnJunegull\fax-0.21\fs150\bord0\shad0\an5}HELLO WORLD
```

**Comment by TheOneric** [comment]
Documenting @astiob’s finding previously shared on IRC:

e50e0002f5567d6005ce61ca8a469a1dae57d10f replaced FreeType’s faux italic (~12 degrees) with our own to match the angle used GDI shown here (~20 degrees).
However as it turns out, GDI’s shear angle actually depends on whether the font uses PostScript or TrueType outlines. The former appears to use a shear angle of ~10 degrees, while the latter matches our new new ~20 degree shear.

**Postscript Outlines:**
![postscript](https://github.com/libass/libass/assets/1668471/e85ddf2f-8211-42ec-85d7-352a64c01264)

**TrueType Outlines:**
![truetype](https://github.com/libass/libass/assets/1668471/f646573c-37b5-43db-a45a-c6b8301c1155)

**Comment by astiob** [comment]
Ah, I _thought_ we already had something on the tracker for this!

This has been fixed by #760.
